app/Http/Requests/Main/ContactUsRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests\Main;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ContactUsRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'txtname' => ['required'],
+            'txtemail' => ['required'],
```

Comment:
Add proper validation like validating the email field, check the name is a string. make limit for store chars like min, max. Also, add validation to check message length.

Reply:
Done Resolved